modules/sdk/project-templates-theme/src/main/resources/archetype-resources/pom.xml

```diff
@@ -54,7 +54,7 @@
 				</executions>
 				<configuration>
 					<docrootDirName>${com.liferay.portal.tools.theme.builder.outputDir}</docrootDirName>
-					<precision>5</precision>
+					<precision>9</precision>
```

Comment:
SOmething weird is going on.  This used to be 9 and then we were seeing faillures and switching it to 5 made it correct.  now we are having to switch it back.  weird.